.rubocop.yml

```diff
@@ -24,7 +24,7 @@ Metrics/MethodLength:
 Style/AccessorMethodName:
   Enabled: false
 
-Style/MethodCallParentheses:
+Style/MethodCallWithoutArgsParentheses:
```

Comment:
This: http://rubocop.readthedocs.io/en/latest/cops_style/#stylemethodcallwithargsparentheses combined with a good `IgnoredMethods` whitelist could technically do the job.